src/libraries/System.Runtime/tests/System/IntPtrTests.GenericMath.cs

```diff
@@ -2647,19 +2647,19 @@ public static void op_LeftShiftTest()
         {
             if (Environment.Is64BitProcess)
             {
-                Assert.Equal(unchecked((nint)0x0000000000000000), ShiftOperatorsHelper<nint, nint>.op_LeftShift(unchecked((nint)0x0000000000000000), 1));
-                Assert.Equal(unchecked((nint)0x0000000000000002), ShiftOperatorsHelper<nint, nint>.op_LeftShift(unchecked((nint)0x0000000000000001), 1));
-                Assert.Equal(unchecked((nint)0xFFFFFFFFFFFFFFFE), ShiftOperatorsHelper<nint, nint>.op_LeftShift(unchecked((nint)0x7FFFFFFFFFFFFFFF), 1));
-                Assert.Equal(unchecked((nint)0x0000000000000000), ShiftOperatorsHelper<nint, nint>.op_LeftShift(unchecked((nint)0x8000000000000000), 1));
-                Assert.Equal(unchecked((nint)0xFFFFFFFFFFFFFFFE), ShiftOperatorsHelper<nint, nint>.op_LeftShift(unchecked((nint)0xFFFFFFFFFFFFFFFF), 1));
+                Assert.Equal(unchecked((nint)0x0000000000000000), ShiftOperatorsHelper<nint, int, nint>.op_LeftShift(unchecked((nint)0x0000000000000000), 1));
+                Assert.Equal(unchecked((nint)0x0000000000000002), ShiftOperatorsHelper<nint, int, nint>.op_LeftShift(unchecked((nint)0x0000000000000001), 1));
+                Assert.Equal(unchecked((nint)0xFFFFFFFFFFFFFFFE), ShiftOperatorsHelper<nint, int, nint>.op_LeftShift(unchecked((nint)0x7FFFFFFFFFFFFFFF), 1));
+                Assert.Equal(unchecked((nint)0x0000000000000000), ShiftOperatorsHelper<nint, int, nint>.op_LeftShift(unchecked((nint)0x8000000000000000), 1));
```

Comment:
I don't see shift mentioned in the API proposal, why is this being changed?

Reply:
This is a test helper. The `shift` API was fixed last week but the test helper didn't get updated at the same time.